lte/gateway/c/core/oai/test/s1ap_task/test_s1ap_handle_new_association.cpp

```diff
@@ -0,0 +1,145 @@
+/**
+ * Copyright 2021 The Magma Authors.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include <gtest/gtest.h>
+#include <glog/logging.h>
+
+extern "C" {
+#include "log.h"
```

Comment:
@electronjoe is there a way to specify the full path here? I tried a few "fuller" paths, but then the header file can't be found

Reply:
Fun... the github `View changes` button takes me to an error that the changes referenced are not found. Maybe due to subsequent forced push (still, seems a design oversight by GH).

Reply:
To answer the question we have to know what the complete list of -I paths are that are being passed to the compiler by the CMake config for the targets that involve this test.  To give a fuller path we would need to specify -I <repo root> or some such - which I was advocating we might want to do /w @themarwhal  as part of Bazel migration simply because it allows us to start fixing up paths everywhere. I think she initially ran into some issues with this, maybe she and I should explore how we get this working generally so we can start migrating paths when desired.